iam_openapi_spec.yml

```diff
@@ -189,6 +189,134 @@ paths:
       security:
         - apiKeyAuth: []
 
+  # sub organizations
+  /organizations/{organization_id}/sub_organizations:
+    post:
+      tags:
+        - Sub Organization Management
+      summary: Create a sub organization under an organization. Sub organization provides a way to create a hierarchy of organizations to easily manage resources at each levels.
+      description: Create a sub organization under an organization.
+      operationId: createSubOrganization
+      parameters:
+        - $ref: '#/components/parameters/organization_id'
+      requestBody:
+        $ref: '#/components/requestBodies/CreateSubOrganizationRequest'
+      responses:
+        '201':
+          $ref: '#/components/responses/CreateSubOrganizationResponse'
+        '400':
+          $ref: '#/components/responses/ErrorResponse'
+        '401':
+          $ref: '#/components/responses/ErrorResponse'
+        '403':
+          $ref: '#/components/responses/ErrorResponse'
+        '404':
+          $ref: '#/components/responses/ErrorResponse'
+        '429':
+          $ref: '#/components/responses/ErrorResponse'
+        default:
+          $ref: '#/components/responses/ErrorResponse'
+
+    get:
+      tags:
+        - Sub Organization Management
+      summary: List sub organizations under an organization.
+      description: List all sub organizations under an organization.
+      operationId: listSubOrganizations
+      parameters:
+        - $ref: '#/components/parameters/organization_id'
+      responses:
+        '200':
+          $ref: '#/components/responses/ListSubOrganizationsResponse'
+        '400':
+          $ref: '#/components/responses/ErrorResponse'
+        '401':
+          $ref: '#/components/responses/ErrorResponse'
+        '403':
+          $ref: '#/components/responses/ErrorResponse'
+        '404':
+          $ref: '#/components/responses/ErrorResponse'
+        '429':
+          $ref: '#/components/responses/ErrorResponse'
+        default:
+          $ref: '#/components/responses/ErrorResponse'
+
+  /organizations/{organization_id}/sub_organizations/{sub_organization_id}:
```

Comment:
/organizations/{organization_id}/sub_organizations/{sub_organization_id}/users/{user_id}

APIs are not present in openapi spec